tests/README.md

```diff
@@ -0,0 +1,90 @@
+MetalK8s is an infrastructure product, that rely on ansible so framework in python is somewhat “obvious”.
+Pytest is quite famous right famous.
+
+Test to be run could be splitted in different category:
+
+1. Test that run before installation on any machine
+    - test of os-check (centos check and kernel)
+    - test first installation of component that doesn't need kubernetes (node_exporter)
```

Comment:
So, test roles in isolation.

Reply:
But could be more than one role, could also be "storage"

Reply:
Shouldn't that ideally be provisioned by one role?

Reply:
Maybe I over developed my ansible "theory",but: no. A role only target one "set" of server. Meaning that we want a set of feature only be developed using one set of server is too restrictive for me (example storage: create storage on each node, but only register on master node)

Reply:
Ah, I see. Fair enough, though most of the time this will involve interaction with a K8s cluster, which will be the 'difficult' roles to test anyway ;)